src/engine/order.spec.ts

```diff
@@ -381,7 +382,7 @@ describe('OrderEngine', () => {
     expect(checkOrder).toHaveBeenCalledTimes(2)
   })
 
-  test('re-execute order if slippage', async () => {
+  test('re-execute order if slippage (buy)', async () => {
```

Comment:
Its nothing new but about the async here:
Are we should that this is working?
I am just somehow worried that the beforeEach and afterEach will not 100% for this.
If i run multiple test-cases in paralell here, they could influence each other, or am i wrong?

p.e:
test case 1 starts --> beforeEach is called
test case 2 starts --> beforeEach is called
test case 1 ends --> afterEach is called ( state of mock functions is resetted)
test case 2 will evaluate and die, because mock is resetted
?
I mean currently we are shipping around this with --runInBand option in jest, but then the async would also be dropped.

Reply:
The `jest` test runner runs each suite (file) in a different process. Also, it's pre-configured to work with promises out of the box and marking a function as `async` automatically makes it return a promise.

Reply:
yeah i am not worried about other test suites, i am worried about the test case. and the problem is not the async itself, its that the live time of the order-enigine and its dependants could change while the test is still in execute.
But as long as we keep the runInBands, this should not be an issue. i tried just to mark that this will be an race condition if its running in parallel.